spec/system/subscriptions_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+describe "Subscriptions" do
+  let(:user) { create(:user, subscriptions_token: SecureRandom.base58(32)) }
+
+  context "Edit page" do
+    scenario "Render content in the user's preferred locale" do
+      user.update!(locale: "es")
+      visit edit_subscriptions_path(token: user.subscriptions_token)
+
+      expect(page).to have_content "Notificaciones"
+      expect(page).to have_field "Recibir un email cuando alguien comenta en mis contenidos", type: :checkbox
+      expect(page).to have_field "Recibir un email cuando alguien contesta a mis comentarios", type: :checkbox
+      expect(page).to have_field "Recibir emails con información interesante sobre la web", type: :checkbox
+      expect(page).to have_field "Recibir resumen de notificaciones sobre propuestas", type: :checkbox
+      expect(page).to have_field "Recibir emails con mensajes privados", type: :checkbox
+      expect(page).to have_button "Guardar cambios"
+    end
+
+    scenario "Use the locale in the parameters when accessing anonymously" do
+      visit edit_subscriptions_path(token: user.subscriptions_token, locale: :es)
+
+      expect(page).to have_content "Notificaciones"
+    end
+  end
+
+  context "Update" do
+    scenario "Allow updating the status notification" do
+      user.update!(email_on_comment: false,
+                   email_on_comment_reply: true,
+                   newsletter: true,
+                   email_digest: false,
+                   email_on_direct_message: true)
+      visit edit_subscriptions_path(token: user.subscriptions_token)
+
+      check "Notify me by email when someone comments on my contents"
+      uncheck "Notify me by email when someone replies to my comments"
+      uncheck "Receive by email website relevant information"
+      check "Receive a summary of proposal notifications"
+      uncheck "Receive emails about direct messages"
+      click_button "Save changes"
+
+      expect(page).to have_content "Changes saved"
+      expect(user.reload.email_on_comment).to eq true
+      expect(user.reload.email_on_comment_reply).to eq false
+      expect(user.reload.newsletter).to eq false
+      expect(user.reload.email_digest).to eq true
+      expect(user.reload.email_on_direct_message).to eq false
```

Comment:
Using `user.reload` means we're accessing the database after the process running the browser has already started, which might result in some `PG::ProtocolViolation` exceptions when running the tests.

How about checking the checkboxes are present with the updated values?